src/main/kotlin/io/github/mojira/arisa/modules/KeepPrivateModule.kt

```diff
@@ -46,11 +46,8 @@ class KeepPrivateModule(
             comment.visibilityValue == "staff" &&
             (comment.body?.contains(keepPrivateTag!!) ?: false)
 
-    private fun isSecurityChange(item: ChangeLogItem) = item.field == "security"
-
-    private fun isSecurityChangeToPublic(item: ChangeLogItem, securityLevel: String?, privateLevel: String) =
-        item.field == "security" && item.changedFromString == privateLevel && item.changedToString == securityLevel &&
-                securityLevel != privateLevel
+    private fun isSecurityChangeToPublic(item: ChangeLogItem, privateLevel: String) =
+        item.field == "security" && item.changedFromString == privateLevel
```

Comment:
Removed two checks:
- `securityLevel != privateLevel`
That is probably redundant; module gets the last changelog entry matching, and also asserts that security level is not public, so (unless Jira is bugged?) `securityLevel != privateLevel` should always be true
- `item.changedFromString == privateLevel`
That is too restrictive; consider the hypothetical case where the security is changed from `private` -> `public` -> `other-public`. Then the module would not do anything because there exists no changelog entry from `private` -> `other-public` (also added a test for this)

Reply:
How about adding a check for `item.changedToString != privateLevel`? Just in case something happens and the security level changes from private to private.

Reply:
Oh wait nevermind, that's also redundant.